app/views/layouts/application.html.erb

```diff
@@ -273,6 +273,7 @@
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Overview", 'integrations/sso' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Okta", 'integrations/sso/okta' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "ADFS", 'integrations/sso/adfs' %></li>
+                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "GitHub", 'github_sso' %></li>
```

Comment:
```suggestion
                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "GitHub", 'github-sso' %></li>
```

I should document this in the README somewhere, but Rails converts underscores in filenames to hypens in links